invoker/core/src/test/java/com/google/cloud/functions/invoker/http/HttpTest.java

```diff
@@ -444,10 +462,11 @@ private void httpResponseEffects(
           response -> response.setStatusCode(HttpStatus.IM_A_TEAPOT_418),
           response -> assertThat(response.getStatus()).isEqualTo(HttpStatus.IM_A_TEAPOT_418)),
       responseTest(
+          // reason string cannot be set by application
```

Comment:
IIUC this comment would be better above line 469 explaining why the assertion is against `Code.IM_A_TEAPOT.getMessage()` and not against `Je suis une théière`